### ISSUE TITLE
[Initials] “A” Looks Too Light Compared to Other Initials

### ISSUE THREAD

**Issue by amarakon** [georgd/EB-Garamond#166]
It appears like the “A” is lighter than all of the other letters for EB Garamond Initials. I’m not sure if this is intentional, but I think it would be better if it was a bit thicker to match the other initials.

![import-2023-03-24_19:36:51](https://user-images.githubusercontent.com/92645199/227670314-997a9820-a518-4a55-874f-b66119f573df.png)

**Comment by timgrei** [comment]
I actually think the B is way too thick.

**Comment by timgrei** [comment]
B and H have a higher gray value than the rest.

**Comment by timgrei** [comment]
Also, if you look at the C and the places where the nib has the same angles as in the A, I think it's reasonably consistent.

**Comment by amarakon** [comment]
> I don't think it needs to be corrected, because the initials are never seen next to each other.

Even though the initials are never next to each other, this inconsistency causes another problem: it is harder to distinguish the A from its background, making it harder to read than other initials, especially at a distance.

**Comment by timgrei** [comment]
We could look through the originals and try to find a thicker A. Then that could be made an alternative. I would be opposed to substantially change a design based on an original.

**Comment by georgd** [comment]
That’s right, A, N and O are considerably lighter than the others and B and somehow also R, T and Y are more on the heavier side. I don’t think it’s bad per se. But adding variants to choose from is definitely a good idea. A model for the logic is in place with the L.01 which I (re-)added recently.

**Comment by timgrei** [comment]
I don't now, if I'm going to do that. The next thing I have planed for the initials is the original K.

**Comment by georgd** [comment]
> I don't know, if I'm going to do that. The next thing I have planned for the initials is the original K.

That’s perfectly fine. Thanks!
